suite-native/module-accounts-import/src/components/AccountImportLoader.tsx

```diff
@@ -1,41 +1,76 @@
-import React from 'react';
+/* eslint-disable global-require */
```

Comment:
nit: Just disable the line of usage if it's only image loading. I think it doesn't need to be turned off for whole file.